### ISSUE TITLE
Shiny web app with shinymeta

### ISSUE THREAD

**Issue by seabbs** [giabaio/BCEA#17]
We discussed how suited your package was to a shiny app. It looks like you have implemented this which is great! 

Would you be interested in implementing [{shinymeta}](https://rstudio.github.io/shinymeta/) so that the code that is being used can be outputted for the user? For some novice R users this could be a really great way of moving from a non-reproducible workflow to a reproducible one!

**Comment by giabaio** [comment]
Yes --- there's a standalone webapp [here](https://egon.stats.ucl.ac.uk/projects/BCEAweb/). Also there's a function `BCEAweb()` inside the latest release of `BCEA` (which you probably have seen), which allows you to open up the web interface to run a `BCEA` analysis. 

I have not worked with `shinymeta` and I think, sadly, it would not be a topic with high priority for me right now --- but certainly I agree with your comment and would be happy to discuss a possible implementation, if you (and possibly some colleagues/teammates?) were interested/had time to work on it!

**Comment by seabbs** [comment]
Awesome - the app looks great. 

Again it would be nice to have this mentioned somewhere in the package docs (perhaps the README) as this seems like such a useful tool. 

I have done some work with `shinymeta` and I am happy to take a look over the next few days. If you leave this issue open I will use it to link a pull request (the shiny code may be difficult to port so it may not be possible).

**Comment by giabaio** [comment]
OK --- let's re-assess soon!
